tests/components/PaginationTest.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { beforeEach, describe, expect, it, vi } from 'vitest';
+
+import { Pagination } from '../../packages/pagination/src/index.js';
+
+const onChangeFunction = vi.fn();
+
+describe('Pagination component', () => {
+    beforeEach(() => {
+        render(<Pagination currentPage={0}
+            numPages={3}
+            lastPage={3}
+            createHref={(page) => `?page=${page}`}
+            onChange={onChangeFunction} />);
+    });
+
+    it('renders', () => {
+        expect(screen.getByText('Pages')).toBeInTheDocument();
+    });
+
+    it('renders an H1', () => {
+        expect(screen.getByRole('heading', { level: 1 })).toBeInTheDocument();
+        expect(screen.getByRole('heading', { level: 1 })).toHaveTextContent('Pages');
```

Comment:
Would it work to chain matchers? 🤔 
```suggestion
        expect(screen.getByRole('heading', { level: 1 })).toBeInTheDocument().and.toHaveTextContent('Pages');
```
...and would it be better/faster? 🤷😄

Reply:
No it doesn't. The counter argument again chaining assertions is that when something fails, it's easier to see which line it fails on. Would be harder to debug when statements are chained, is it the first one or the second one?

Reply:
Agreed. 🙂👍  ...but then, how about this? 😄 
```suggestion
        const heading = screen.getByRole('heading', { level: 1 });
        expect(heading).toBeInTheDocument();
        expect(heading).toHaveTextContent('Pages');
```

Reply:
Sure 👍